app/cookies/CookieType.scala

```diff
@@ -0,0 +1,40 @@
+package cookies
+
+import play.api.mvc.Cookie
+
+abstract class CookieType {
```

Comment:
as we talked about, I got confused by the name of `CookieType`, not realising it was the definition of a cookie rather than containing the value of the cookie. I think it could be renamed to something more expllicit, including it sub-class `ContribTimestampCookie`